src/main/java/pl/agh/edu/master_diet/core/model/database/Browsable.java

```diff
@@ -0,0 +1,27 @@
+package pl.agh.edu.master_diet.core.model.database;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@MappedSuperclass
+@NoArgsConstructor
+@AllArgsConstructor
+public abstract class Browsable {
```

Comment:
Naming abstract classes with "able" is not a Java convention

Reply:
this was interface first. Changed.